starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/repository.rocker.raw

```diff
@@ -10,4 +10,4 @@
     <enabled>false</enabled>
   </releases>
 }
-</repository>
\ No newline at end of file
+</repository>
```

Comment:
I added a blank line to the end here as it makes the POM nicer

I know XML is for machines to read, but we have to look at it too ;-)